lab/login/style.css

```
@@ -280,3 +280,24 @@
 .name-R74n::after {
   content: " 🔒";
 }
+
```

```suggestion
.at-Tuna::after {
  all: unset;
}
```

I will merge if we remove the tag from mentions, which can get a bit long :)